pkg/events/definitions.go

```diff
@@ -71,7 +72,7 @@ func defineSkip(name, description string, skip uint, opts ...Option) Builder {
 	i18n.Define(fmt.Sprintf("%s:%s", i18nPrefix, name), description).SetSource(1 + skip)
 	initMetrics(name)
 
-	var b Builder = def
+	b := def.With(defaultOptions...)
```

Comment:
IMO `defaultOptions` should not be applied here. Since this is the only way to construct an event and these are always applied, it makes no sense to have them as options - all side effects should be directly applied at construction time.

Reply:
cc @htdvisser, since this is related to changes in #3018.

Reply:
If it makes @rvolosatovs feel better we can rename these things "event mutators" or something.